File: desapp-groupB-backend/src/main/java/dto/PublicationRequestDto.java
package dto;

import model.booking.BookingRequest;
import model.publication.Publication;

public class PublicationRequestDto {
	
	public Publication getPublication() {
		return publication;
	}
	public void setPublication(Publication publication) {
		this.publication = publication;
	}
	public BookingRequest getRequest() {
		return request;
	}
	public void setRequest(BookingRequest request) {
		this.request = request;
	}
	private Publication publication;
	private BookingRequest request;

}

File: carpndWeb/src/app/models/publication-request-dto.ts
import { Publication } from './../models/publication';
import { BookingRequest } from './../models/booking-request';

export class PublicationRequestDto {
	publication: Publication;
	request: BookingRequest;

	constructor(publication: Publication, request: BookingRequest){
		this.publication = publication;
		this.request = request;
	}
}
